b9lab-content/4-running-a-chain/04-node-api-and-cli.md

```diff
@@ -1 +1,292 @@
+
+```
+
+Now, start your blockchain:
+
+```bash
+$ ./simd start
+
+6:23PM INF starting ABCI with Tendermint
+6:23PM INF Starting multiAppConn service impl=multiAppConn module=proxy
+6:23PM INF Starting localClient service connection=query impl=localClient module=abci-client
+6:23PM INF Starting localClient service connection=snapshot impl=localClient module=abci-client
+6:23PM INF Starting localClient service connection=mempool impl=localClient module=abci-client
+6:23PM INF Starting localClient service connection=consensus impl=localClient module=abci-client
+
+```
+
+You will see blocks producing and validating. Open a new terminal and check the balances:
+
+```bash
+$ ./simd query bank balances $(./simd keys show b9lab -a)
+balances:
+- amount: "30000000"
+  denom: stake
+pagination:
+  next_key: null
+  total: "0"
+```
+
+Create another key for a student and send it some tokens:
```

Comment:
```suggestion
Create another account for a student and send that account some tokens:
```

Reply:
oh, are we using "key" or "account" (I think account is more commonly used in all the docs and tutorials to date) let's always use the same term, we explained key and account were the same earlier in this document